runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -70,6 +106,64 @@ public void updateClanChatTitle()
 		}
 	}
 
+	private boolean send(ChatboxInput chatbox)
+	{
+		executor.execute(() ->
+		{
+			chatbox.resume();
```

Comment:
You dont need to .resume if you never called .pause

Reply:
Resume sends the chat message. See, https://github.com/runelite/runelite/blob/9fe667c763e6f82d218da00a12aaf051d0defaa9/runelite-client/src/main/java/net/runelite/client/chat/CommandManager.java#L131.

It's an abstract class that is created when the event is dispatched.